e2e/tests/smoke_test.js

```diff
@@ -1,11 +1,7 @@
 /* global process */
-// const config = require('../config.js');
 
 Feature('Smoke tests @smoke-tests');
```

Comment:
Worth discussing maybe our smoke tests should at least create a case, we would ensure our pods are started properly and not only ccd, but out of scpoe here.

Reply:
yep, but it is not this ticket